src/rainfly/src/lib/components/nav/items/NavExamples.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import NavItem from '$lib/components/nav/NavItem.svelte';
+  import NavDropdownItem from '$lib/components/nav/NavDropdownItem.svelte';
+  import {fetchTextFile} from '$lib/utils/file-utils.js';
+  import {
+    loadEditorProcessor,
+    loadEditorMain,
+  } from '$lib/../routes/+page.svelte';
+
+  /**
+   * Load processor and main code from URL to Editor
+   * @param {string} mainCodeUrl - URL to main code
+   * @param {string} processorCodeUrl - URL to processor code
+   */
+  async function loadExample(mainCodeUrl, processorCodeUrl) {
+    loadEditorMain((await fetchTextFile(mainCodeUrl)).data, 'main');
+    loadEditorProcessor(
+        (await fetchTextFile(processorCodeUrl)).data,
+        'processor',
+    );
+  }
+</script>
+
+<NavItem name="Examples">
```

Comment:
So the example list is hardcoded for now?

Reply:
It was easy enough to convert to a `examples.json` file so I did that instead :D